data/transactions/logic/doc.go

```diff
@@ -337,6 +337,7 @@ var txnFieldDocs = map[string]string{
 	"VoteFirst":                "The first round that the participation key is valid.",
 	"VoteLast":                 "The last round that the participation key is valid.",
 	"VoteKeyDilution":          "Dilution for the 2-level participation key",
+	"Nonparticipation":         "Key registration participation flag",
```

Comment:
How about "Marks an account nonparticipating for rewards"?

That is similar to the language we use here: https://developer.algorand.org/docs/reference/transactions/